pkg/metrics/metrics.go

```diff
@@ -22,6 +22,13 @@ import (
 	"github.com/openshift/windows-machine-config-operator/pkg/nodeconfig"
 )
 
+//+kubebuilder:rbac:groups="",resources=services;services/finalizers,verbs=create;get;delete
+//+kubebuilder:rbac:groups="",resources=endpoints,verbs=create;get;delete;update;patch
+//+kubebuilder:rbac:groups="",resources=namespaces,verbs=get
+//+kubebuilder:rbac:groups="",resources=nodes,verbs=list
+//+kubebuilder:rbac:groups=monitoring.coreos.com,resources=servicemonitors,verbs=get;list;delete
+//+kubebuilder:rbac:groups="",resources=events,verbs=*
+
```

Comment:
Aren't we missing the RBAC from [windows-exporter-role.yaml](https://github.com/openshift/windows-machine-config-operator/blob/master/deploy/olm-catalog/windows-machine-config-operator/manifests/windows-exporter-role.yaml)